frontend/public/components/pod-exec.jsx

```diff
@@ -140,7 +146,20 @@ export class PodExec extends React.PureComponent {
   }
 
   render () {
-    const {containers, activeContainer } = this.state;
+    const { containers, activeContainer, open, error } = this.state;
+
+    let contents;
+    if (error) {
+      contents = <div className="text-center cos-error-title">{error}</div>;
```

Comment:
I wouldn't block the PR over this, but it'd be nice to keep the terminal contents around even when there's an error. I could see users being annoyed when the text they were just looking at gets replaced with an error message.